bridges/snowbridge/pallets/outbound-queue-v2/src/test.rs

```diff
@@ -308,3 +309,54 @@ fn encode_register_pna() {
 	println!("{}", HexDisplay::from(&message_abi_encoded));
 	assert_eq!(hex!("000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000003e80000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000124f80000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000").to_vec(), message_abi_encoded)
 }
+
+#[test]
+fn process_delivery_receipt_with_reward_address() {
```

Comment:
please add this test to Ron's PR/fix

Reply:
https://github.com/yrong/polkadot-sdk/pull/17 cc @yrong